docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -101,7 +101,7 @@ when the pod is created, so it is ignored by the scheduler).  Therefore:
 Daemon pods do respect [taints and tolerations](/docs/concepts/configuration/assign-pod-node/#taints-and-tolerations-beta-feature),
 but they are created with `NoExecute` tolerations for the following taints with no `tolerationSeconds`:
 
- - `node.alpha.kubernetes.io/notReady`
+ - `node.alpha.kubernetes.io/not-ready`
  - `node.alpha.kubernetes.io/unreachable`
  - `node.alpha.kubernetes.io/memoryPressure`
```

Comment:
s/memoryPressure/memory-pressure

Reply:
yes, when the time comes to fix the doc, we will need to change diskPressure, outOfDisk as well.
Also this patch has better be rebased on #5048 which is moving taint and toleration into a separate doc.